service/deploy.py

```diff
@@ -158,10 +158,22 @@ def configure_ansible(logger):
 
 
 def build_host_name(ip):
+    """
+    Return the host name
+    * iPlant or raw
```

Comment:
If we ever lost DNS and couldn't keep `vmX-Y` as a value? Probably never.. really keeping them both around to make swapping back to jetstream easier (if necessary)